docs/DeveloperGuide.md

```diff
@@ -1066,8 +1066,30 @@ testers are expected to do more *exploratory* testing.
 
 #### View student information
 
-1. _{ more test cases to come …​ }_
-
+1. View all related information of a student
```

Comment:
Might want to be more specific about this test case, e.g is a student currently being viewed in the Result Display? Or is the Result Display blank (startup)